### ISSUE TITLE
To-do list check marks fail WCAG

### ISSUE THREAD

**Issue by oleq** [ckeditor/ckeditor5#3024]
![image](https://user-images.githubusercontent.com/1099479/63262022-34930500-c285-11e9-863a-b01276c0c891.png)

https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=30D23F

[Should be #1D8727 to pass AA for normal text](https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=1D8727).

![image](https://user-images.githubusercontent.com/1099479/63262153-80de4500-c285-11e9-8c12-cf359b5b5c1b.png)

cc @dkonopka
